client/src/components/onePirate/SignIn.js

```diff
@@ -1,117 +1,122 @@
+              t.palette.mode === 'light' ? t.palette.grey[50] : t.palette.grey[900],
+            backgroundSize: 'cover',
+            backgroundPosition: 'center',
+          }}
+        />
+        <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+          <Box
+            sx={{
+              my: 8,
+              mx: 4,
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+            }}
+          >
+            <Avatar sx={{ m: 1, bgcolor: 'secondary.main' }}>
+              <LockOutlinedIcon />
+            </Avatar>
+            <Typography component="h1" variant="h5">
+              Sign in
+            </Typography>
+            <Box component="form" noValidate onSubmit={handleSubmit} sx={{ mt: 1 }}>
+              <TextField
+                margin="normal"
+                required
+                fullWidth
+                id="email"
+                label="Email Address"
+                name="email"
                 autoComplete="email"
                 autoFocus
-                component={RFTextField}
-                disabled={submitting || sent}
-                fullWidth
-                label="Email"
+              />
+              <TextField
                 margin="normal"
-                name="email"
                 required
-                size="large"
-              />
-              <Field
                 fullWidth
-                size="large"
-                component={RFTextField}
-                disabled={submitting || sent}
-                required
                 name="password"
-                autoComplete="current-password"
                 label="Password"
                 type="password"
-                margin="normal"
+                id="password"
+                autoComplete="current-password"
               />
-              <FormSpy subscription={{ submitError: true }}>
-                {({ submitError }) =>
-                  submitError ? (
-                    <FormFeedback error sx={{ mt: 2 }}>
-                      {submitError}
-                    </FormFeedback>
-                  ) : null
-                }
-              </FormSpy>
-              <FormButton
-                sx={{ mt: 3, mb: 2 }}
-                disabled={submitting || sent}
-                size="large"
-                color="secondary"
+              <FormControlLabel
+                control={<Checkbox value="remember" color="primary" />}
+                label="Remember me"
+              />
+              <Button
+                type="submit"
                 fullWidth
+                variant="contained"
+                sx={{ mt: 3, mb: 2 }}
               >
-                {submitting || sent ? 'In progress…' : 'Sign In'}
-              </FormButton>
+                Sign In
+              </Button>
+              <Grid container>
+                <Grid item xs>
+                  <Link href="#" variant="body2">
```

Comment:
This Link doesn't navigate anywhere, which I'm sure is expected and known but I might suggest adding a TODO comment here so you can search later for the things that need continued work and are expected to not work quite yet.

Reply:
Added TODO's to this link as well as the actual submit button